chromium_src/components/component_updater/component_installer.cc

```diff
@@ -1,8 +1,48 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/component_updater/component_installer.h"
 
 #include "components/crx_file/crx_verifier.h"
 #define CRX3_WITH_PUBLISHER_PROOF CRX3
+#define Register Register_ChromiumImpl
 #include "../../../../components/component_updater/component_installer.cc"
+#undef Register
 #undef CRX3_WITH_PUBLISHER_PROOF
+
+namespace component_updater {
+
+void ComponentInstaller::Register(ComponentUpdateService* cus,
+                                  base::OnceClosure callback) {
+  static const char* blacklisted_components[] = {
```

Comment:
Sorry, didn't notice before. I believe, there is an effort in Chromium and elsewhere to transition to using `allow/disallow` terminology.

Reply:
No problem :) I'll update.

Reply:
Fixed in 49406522.